extensions/ql-vscode/src/history-item-label-provider.ts

```diff
@@ -60,4 +62,20 @@ export class HistoryItemLabelProvider {
       '%': '%',
     };
   }
+
+  private getRemoteInterpolateReplacements(item: RemoteQueryHistoryItem): InterpolateReplacements {
+    return {
+      t: new Date(item.remoteQuery.executionStartTime).toLocaleString(env.language),
+      q: item.remoteQuery.queryName,
+
+      // There is no database name for remote queries. Instead use the controller repository name.
+      d: `${item.remoteQuery.controllerRepository.owner}/${item.remoteQuery.controllerRepository.name}`,
+
+      // There is no synchronous way to get the results count.
+      r: '',
```

Comment:
I think it's fine to have the result count empty, but perhaps we should also change the default string? It looks a bit strange to have `   result count` in the default label, even for remote queries:

![image](https://user-images.githubusercontent.com/42641846/163167676-bd8acecd-9c71-4ada-820a-6d49e69ead4c.png)

Reply:
Yes. That does look odd. Not sure what to put there...